modules/ding_ting_frontend/ding_ting_frontend.features.field_instance.inc

```diff
@@ -241,16 +241,18 @@ function ding_ting_frontend_field_default_field_instances() {
         'weight' => 11,
       ),
       'search_result' => array(
-        'label' => 'above',
+        'label' => 'hidden',
```

Comment:
This change is not related as well. This is because you have another module enabled.